packages/runner/src/builtins/compile-and-run.ts

```diff
@@ -85,7 +85,7 @@ export function compileAndRun(
     // Now we're sure that we have a new file to compile
     compiling.setAtPath([], true, log);
 
-    const compilePromise = runtime.harness.runSingle(
+    const compilePromise = runtime.harness.run(
```

Comment:
(to myself or whoever knows): do we have test coverage for compileAndRun builtin?  should we include it in integrations?

Reply:
we don't yet have that. we should.